docs/FAQ.md

```diff
@@ -5,8 +5,8 @@
 ```{toctree}
 :maxdepth: 3
 
-https://github.com/reactive-firewall/multicast/.github/CODE_OF_CONDUCT.md
-https://github.com/reactive-firewall/multicast/.github/CONTRIBUTING.md
+[Code of Conduct](https://github.com/reactive-firewall/multicast/.github/CODE_OF_CONDUCT.md)
+[Contributing](https://github.com/reactive-firewall/multicast/.github/CONTRIBUTING.md)
```

Comment:
🤔 I might be mistaken but I don't see much of a *compelling* reason to pin the url to the stable branch instead of the default branch (especially considering these guidelines are more relevant to the default development branch than the release branch in general) 🤷🏻